Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -49,9 +44,28 @@ public override void OnPreSceneQuery()
                 Rays = new RayStep[1];
             }
 
+            //Proximity check
+            BaseNearInteractionGrabbable newClosestGrabbable = null;
             Vector3 pointerPosition;
+
             if (TryGetNearGraspPoint(out pointerPosition))
             {
+                closestDistance = float.PositiveInfinity;
+                foreach (BaseNearInteractionGrabbable prox in BaseNearInteractionGrabbable.Instances)
+                {
+                    if (prox.ColliderEnabled)
+                    {
+                        float dist = prox.DistanceToGrabbable(pointerPosition);
+                        if (dist < SphereCastRadius + 0.05f && dist < closestDistance)
```

Comment:
I guess it's some kind of margin? Does it make sense to make it configurable? (i know you didn't add it, but any idea what this is for?)

Reply:
I'm assuming it exists to bump up the inaccuracy of someone targeting an object to pick it up - i.e. they may mis-grab slightly. That said, SphereCastRadius could just be 0.05f larger 😄

Reply:
Now that @julenka showed me the Physics Raycasting bits in the `FocusProvider`, I better understand why this 0.05f margin exists. `SpherePointer` flags itself as "near" a collider to let `FocusProvider` do the more expensive overlap check.